text/text.go

```diff
@@ -300,3 +302,51 @@ func Draw(dst *ebiten.Image, text string, face font.Face, x, y int, clr color.Co
 
 	textM.Unlock()
 }
+
+// MeasureString measures the size of a given string using a given font.
+//
+// text is the string that's being measured.
+// face is the font for text rendering.
+//
+// Be careful that the passed font face is held by this package and is never released.
+// This is a known issue (#498).
+//
+// MeasureString is concurrent-safe.
+func MeasureString(text string, face font.Face) (int, int) {
+	textM.Lock()
+
+	x, y := 0, 0
+	w, h := 0, 0
+
+	m := face.Metrics()
+	faceHeight := m.Height
+	faceDescent := m.Descent
+
+	fx, fy := fixed.I(x), fixed.I(y)
+	prevR := rune(-1)
+
+	runes := []rune(text)
+
+	for _, r := range runes {
+		if prevR >= 0 {
+			fx += face.Kern(prevR, r)
+		}
+		if r == '\n' {
+			fx = fixed.I(x)
+			fy += faceHeight
+			prevR = rune(-1)
+			continue
+		}
+
+		fx += glyphAdvance(face, r)
+
+		w = int(math.Max(float64(w), float64(fx.Round())))
+		h = int(math.Max(float64(h), float64(fy.Round() + faceHeight.Round())))
+
+		prevR = r
+	}
+
+	textM.Unlock()
+
+	return w, h + faceDescent.Round()
+}
```

Comment:
Add a new line. (gofmt should do this :-)

Reply:
I swear I ran gofmt on this. 😅 
Will do.

Reply:
A newline was added in 481985a.